src/components/modal/__tests__/__snapshots__/modal.test.js.snap

```diff
@@ -0,0 +1,395 @@
+      aria-describedby="pf-modal-part-4"
+      aria-label="t(modal.aria-label-default)"
+      aria-labelledby="pf-modal-part-2"
+      aria-modal="true"
+      class="pf-c-modal-box pf-m-align-top"
+      data-ouia-component-id="OUIA-Generated-Modal-3"
+      data-ouia-component-type="PF4/ModalContent"
+      data-ouia-safe="true"
+      id="pf-modal-part-2"
+      role="dialog"
+      style="--pf-c-modal-box--m-align-top--spacer: 5%;"
+    >
+      <header
+        class="pf-c-modal-box__header"
+      >
+        lorem ipsum
+      </header>
+      <div
+        aria-live="polite"
+        class="pf-c-modal-box__body"
+        id="pf-modal-part-4"
+      >
+        <div>
+          hello world
+        </div>
+      </div>
+      <footer
+        class="pf-c-modal-box__footer"
+      >
+        dolor sit
+      </footer>
+    </div>
+  </div>
+</div>
+`;
+
+exports[`Modal Component should allow custom headers and footers: string 1`] = `
+<div
+  class="pf-c-backdrop"
+>
+  <div
+    class="pf-l-bullseye"
+  >
+    <div
+      aria-describedby="pf-modal-part-4"
+      aria-label="t(modal.aria-label-default)"
+      aria-labelledby="pf-modal-part-2"
+      aria-modal="true"
+      class="pf-c-modal-box pf-m-align-top"
+      data-ouia-component-id="OUIA-Generated-Modal-3"
+      data-ouia-component-type="PF4/ModalContent"
+      data-ouia-safe="true"
+      id="pf-modal-part-2"
+      role="dialog"
+      style="--pf-c-modal-box--m-align-top--spacer: 5%;"
+    >
+      <header
+        class="pf-c-modal-box__header"
+      >
+        lorem ipsum
+      </header>
+      <div
+        aria-live="polite"
+        class="pf-c-modal-box__body"
+        id="pf-modal-part-4"
+      >
+        <div>
+          hello world
+        </div>
+      </div>
+      <footer
+        class="pf-c-modal-box__footer"
+      >
+        dolor sit
+      </footer>
+    </div>
+  </div>
+</div>
+`;
+
+exports[`Modal Component should allow custom headers and footers: undefined 1`] = `
+<div
+  class="pf-c-backdrop"
+>
+  <div
+    class="pf-l-bullseye"
+  >
+    <div
+      aria-describedby="pf-modal-part-4"
+      aria-label="t(modal.aria-label-default)"
+      aria-labelledby="pf-modal-part-2"
+      aria-modal="true"
+      class="pf-c-modal-box pf-m-align-top"
+      data-ouia-component-id="OUIA-Generated-Modal-3"
+      data-ouia-component-type="PF4/ModalContent"
+      data-ouia-safe="true"
+      id="pf-modal-part-2"
+      role="dialog"
+      style="--pf-c-modal-box--m-align-top--spacer: 5%;"
+    >
+      <div
+        aria-live="polite"
+        class="pf-c-modal-box__body"
+        id="pf-modal-part-4"
+      >
+        <div>
+          hello world
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+`;
+
+exports[`Modal Component should allow modifying specific and custom props: aria-label 1`] = `
+<Modal
+  actions={Array []}
+  appendTo={
+    <div
+      class="quipucords-modal quipucords-modal__hide-backdrop quipucords-modal__rcue-width"
```

Comment:
required:
component not updating on this test, `hide-backdrop` shouldn't be there